### DISCUSSION TITLE
Comma when using lm_optimize?

### DISCUSSION THREAD

**Discussion by FrontierDK** [coqui-ai/STT#2261] [Q&A] [answered]
Hi all :)

Is there a limitation on what can be used in the sample.csv file? I am trying to use strings with a comma, say something like this:

**wav_filename,wav_filesize,transcript**
files/yes_this_is_a_test.wav,94140,yes, this is a test

Also, I add the exact same line to language_model.txt

I can run both the generate_lm.py and generate_scorer_package - but when it gets to lm_optimize - I get this error:
_TypeError: sequence item 2: expected str instance, float found_

How does one use a comma in the text to be learned from? If I remove the comma, all runs well.

**Comment by reuben** [comment, marked as answer]
Add quotes around the transcript:
```csv
 files/yes_this_is_a_test.wav,94140,"yes, this is a test"
```

**Reply by FrontierDK** [reply]
Thank you VERY much 👍
